### DISCUSSION TITLE
813

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#986] [Comments]
# 813

4개월간 진행된 PM 직군의 채용이 마무리 되었다.원래 TO는 1명이였지만 너무 훌륭한 분들이 많이 지원해주셔서 2분을 뽑게 되었다.최종 합격한 2분 외에도 훌륭한 분들이 많으셔서 다음에 기회가 된다면 또 뵙고 싶다는 이야기를 전달드렸다.개발직군에 비해 PM 직군은 지원자수가 많지 않기에 최대한 많은 분들에게 전달하고자 랠릿 외에도 타 플랫폼도 함께 사용했다.2개의 채용 서비스를 이용한 채용은 전체 161분이 지원해주셨고 그 결과 수치는 다음과 같다.161분의 지원자 중 서류는 총 20분이 합격되셨고, 이 중 최종 합격이 2분이 되셨다.많은 분들이 지원해주시고, 그 안에서 면접 경험도 대단히 훌륭해서 기분좋은 채용 과정이였지만, 한가지 우리를 힘들게 했던 것이 있었다.그건 바로 (지원자에 의한) 높은 면..

https://jojoldu.tistory.com/813

**Comment by rainingdaewoo** [comment]
이제 지원자들의 부담은 더 늘어나겠네요
